### PR TITLE
Basic webserver timeline

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Made basic verson of the webserver timeline

### CODE REVIEW

File: src/main/webserver/routes/timeline.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Created by Bart on 17/05/2016.
+ */
```
Comment (geenen124) [inline src/main/webserver/routes/timeline.js]: Replace this with gotofail or remove it?

Comment (bart-de-jonge) [inline src/main/webserver/routes/timeline.js]: yokay

**Comment by Markvanderuit** [comment]
Looks fine, and all commentary is fixed.

**Comment by geenen124** [comment]
Looks good!

**Pull request merged**